datacapturing/src/androidTest/java/de/cyface/datacapturing/model/CapturedDataWriterTest.java

```diff
@@ -173,9 +173,6 @@ public void testCreateNewMeasurement() throws NoSuchMeasurementException, Cursor
                     is(equalTo(Vehicle.UNKNOWN.getDatabaseIdentifier())));
             assertThat(result.getString(result.getColumnIndex(MeasurementTable.COLUMN_STATUS)),
                     is(equalTo(MeasurementStatus.OPEN.getDatabaseIdentifier())));
-            assertThat(result.getInt(result.getColumnIndex(MeasurementTable.COLUMN_ACCELERATIONS)), is(equalTo(0)));
```

Comment:
we don't store the sensor point counters in the measurement entry (database) anymore but just calculate them before synchronization in the temporary transfer binary